examples/composer-shared-vpc/composer_v1_pvt_shared_vpc/variables.tf

```diff
@@ -0,0 +1,156 @@
+  default     = ""
+}
+
+variable "zone" {
+  description = "Zone where the Cloud Composer nodes are created."
+  type        = string
+  default     = "us-central1-f"
+}
+
+variable "node_count" {
+  description = "Number of worker nodes in Cloud Composer Environment."
+  type        = number
+  default     = 3
+}
+
+variable "machine_type" {
+  description = "Machine type of Cloud Composer nodes."
+  type        = string
+  default     = "n1-standard-8"
+}
+
+variable "composer_service_account" {
+  description = "Service Account for running Cloud Composer."
+  type        = string
+  default     = null
+}
+
+variable "composer_service_account_create" {
+  description = "Create composer service account"
+  type        = bool
+  default     = true
+}
+
+variable "disk_size" {
+  description = "The disk size for nodes."
+  type        = string
+  default     = "100"
+}
+
+variable "tags" {
+  description = "Tags applied to all nodes. Tags are used to identify valid sources or targets for network firewalls."
+  type        = set(string)
+  default     = []
+}
+
+
+variable "pod_ip_allocation_range_name" {
+  description = "The name of the cluster's secondary range used to allocate IP addresses to pods."
+  type        = string
+  default     = null
+}
+
+variable "service_ip_allocation_range_name" {
+  type        = string
+  description = "The name of the services' secondary range used to allocate IP addresses to the cluster."
+  default     = null
+}
+
+variable "cloud_sql_ipv4_cidr" {
+  description = "The CIDR block from which IP range in tenant project will be reserved for Cloud SQL."
+  type        = string
+  default     = null
+}
+
+variable "web_server_ipv4_cidr" {
+  description = "The CIDR block from which IP range in tenant project will be reserved for the web server."
+  type        = string
+  default     = null
+}
+
+variable "master_ipv4_cidr" {
+  description = "The CIDR block from which IP range in tenant project will be reserved for the master."
+  type        = string
+  default     = null
+}
+
+variable "web_server_allowed_ip_ranges" {
+  description = "The network-level access control policy for the Airflow web server. If unspecified, no network-level access restrictions will be applied."
+  default     = null
+  type = list(object({
+    value       = string,
+    description = string
+  }))
+}
+
+variable "firewall_rules_create" {
+  description = "Create Egress firewall rules for composer env, needed where egress is denied"
+  type        = bool
+  default     = true
+}
+
+variable "composer_sa_permissions" {
+  description = "IAM Roles assigned to composer SA"
+  type        = list(string)
+  default = [
+    "roles/composer.worker",
+    "roles/iam.serviceAccountUser",
+    "roles/logging.logWriter"
+  ]
+}
+
+variable "assign_robot_sa_permissions" {
+  description = "Assign permissions to GKE and Composer robot agent service accpunts for Shared VPC"
```

Comment:
Do we really need this flag? If we set it to false the Composer deployment will fail ...

Reply:
Removed this config.